theano/sandbox/multinomial.py

```diff
@@ -45,18 +46,18 @@ def make_node(self, pvals, unis):
         else:
             odtype = self.odtype
         out = T.tensor(dtype=odtype, broadcastable=pvals.type.broadcastable)
-        return Apply(self, [pvals, unis], [out])
+        return Apply(self, [pvals, unis, as_scalar(n)], [out])
 
     def grad(self, ins, outgrads):
-        pvals, unis = ins
+        pvals, unis, n = ins
         (gz,) = outgrads
         return [T.zeros_like(x) for x in ins]
 
     def c_code_cache_version(self):
-        return (5,)
+        return None
```

Comment:
Setting it to 'None' is fine for development but you need to set if back to a reasonnable value like '(6,)' or else your PR will deactivate the C code caching for this op.